docs/guide/improving-performance.md

```diff
@@ -85,6 +85,8 @@ vitest run --reporter=blob --shard=2/3 # 2nd machine
 vitest run --reporter=blob --shard=3/3 # 3rd machine
 ```
 
+> Vitest splits your _test files_, not your test cases, into shards. If you've got 1000 test files, the `--shard=1/4` option will run 250 test files, no matter how many test cases individual files have.
```

Comment:
Putting this as a sidenote because it's more meaningful once you've seen the usage example and understand how to apply sharding.